### PR TITLE
Factor out show_vcc from bmct

### PR DESCRIPTION
**Pull request by peterschrammel**
~Based on #2883~

~Only review last commit~

Could also be moved to goto-symex [done]

### CODE REVIEW

**Review by tautschnig — CHANGES_REQUESTED** [review]
+1 for moving it to goto-symex - the code depends on an implementation detail (symex_target_equation.h) of goto-symex.

**Comment by peterschrammel** [comment]
@tautschnig, moved.

**Review by tautschnig — APPROVED** [review]

File: unit/Makefile [hunk 1]
```diff
@@ -69,7 +69,6 @@ BMC_DEPS =../src/cbmc/all_properties$(OBJEXT) \
           ../src/cbmc/cbmc_solvers$(OBJEXT) \
           ../src/cbmc/counterexample_beautification$(OBJEXT) \
           ../src/cbmc/fault_localization$(OBJEXT) \
-          ../src/cbmc/show_vcc$(OBJEXT) \
```
Comment (mgudemann) [inline unit/Makefile:71]: is the moved one not needed here ?

Comment (tautschnig) [reply unit/Makefile:71]: The moved one is dragged in implicitly via `goto-symex$(LIBEXT)`.

**Review by mgudemann — APPROVED** [review]

**Pull request merged**